packages/ffe-grid/less/ffe-grid.less

```diff
@@ -21,141 +21,538 @@
 }
 
 .ffe-grid__row {
-    padding-right: @ffe-spacing-sm;
-    padding-left: @ffe-spacing-sm;
+    padding-right: var(--ffe-spacing-sm);
```

Comment:
var jo ikke det du endret nå, men til neste gang kanskje, vet du om padding-block og padding-inline? 😄 Så slipper man to linjer... Hva er forresten grunnen til å gå bort fra less-variabler og bruke css-variabler i stedet?

Reply:
Måtte kanskje definert writing-mode eller direction i tillegg da eller? Jeg vet ikke, har ikke tenkt så mye på hvorfor padding ikke er slått sammen til shorthand må jeg innrømme. 😅 

CSS-variabler er støttet native i browserne nå for tida, i motsetning til Less. Så da slipper vi i utgangspunktet å håndtere det i byggesteg, bl.a.

Reply:
Gott poeng med shorthand. Mange linjer og spare men trengs ju ikke gjøras her da før min del iaf